db/migrate/20220722101202_add_reference_to_planning_applications.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AddReferenceToPlanningApplications < ActiveRecord::Migration[6.1]
+  def up
+    add_column :planning_applications, :reference, :string
+    add_index :planning_applications, :reference
```

Comment:
I guess we have a unique index on the application number / council code but wonder if we want to add it here too 🤔

Reply:
I guess it would never flag up because of the other unique index though

Reply:
Might be nice just for completeness!